SU-SWS/stanford_basic
# D8CORE-2944: Using flex for the alignment

# READY FOR REVIEW

# Summary
- Getting the card version Events and People grid to align to columns

# Review By (Date)
- 2/24

# Urgency
- Med

# Steps to Test

1. pull in this change.
2. clear caches.
3. create 5 peoples
4. Verify on the filter page, they align in columns evenly.
5. create some events.
6. Create some paragraphs with Card Events and see they align in columns evenly.
7. Create some paragraphs with people and see they align in columns evenly.

# Affected Projects or Products
- SB theme

# Associated Issues and/or People
-  D8CORE-2944

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
